irods/helpers/__init__.py

```diff
@@ -141,3 +141,4 @@ def get_collection(sess, logical_path):
         return sess.collections.get(logical_path)
     except ex.CollectionDoesNotExist:
         return None
+
```

Comment:
Is the addition of this empty line needed/intentional?

Reply:
not there anymore after rebase